x-pack/plugins/uptime/public/components/overview/monitor_list/monitor_list.tsx

```diff
@@ -116,9 +120,9 @@ export const MonitorListComponent: React.FC<Props> = ({
     },
     {
       align: 'left' as const,
-      field: 'state.tls',
+      field: 'state.tls.server.x509',
       name: labels.TLS_COLUMN_LABEL,
-      render: (tls: any) => <CertStatusColumn cert={tls?.[0]} />,
+      render: (x509: any) => <CertStatusColumn expiry={x509} />,
```

Comment:
also it seems like from the code, with this change monitor list cert status column will not work for older data pre 7.8. I will test it after code review.

Reply:
I think you're right, good point. That said, I'm OK with that given that cert monitoring was introduced in 7.8. Supporting older versions requires more code, and there's a cost to supporting that. I'm not sure it's worth the tradeoff / complexity. While we should maintain compatibility with old versions, supporting new features against old versions isn't something we guarantee.